pyproject.toml

```diff
@@ -40,10 +40,10 @@ readme = 'README.md'  # Markdown files are supported
 include = [".git", "indica/git_version"]
 
 [tool.poetry.dependencies]
-python = "^3.6.1"
+python = "^3.8"
```

Comment:
Could we cope with ^3.7? I'm not sure Heimdall has 3.8.

Reply:
Unfortunately not, there are too many features and dependencies in the code that rely on python 3.8. Even though the requirement in the file was 3.6.1 before, all CI tests were run on python 3.8. An example of the features seems to be unpacking operations inside a tuple ie. a return statement that returns a tuple which has a unpack operation for one of its elements - see line 499 in `indica/operators/invert_radiation.py` but there are other bits as well (like specific types from `typing`). I think @elitherl is running InDiCA on Heimdall so I guess he could guide you through setting up a version python 3.8.

Reply:
Yep, I'm running on heimdall with a python 3.9.10 install. I've found [pyenv](https://github.com/pyenv/pyenv) to be the most convenient way to maintain user installs of different python version, they also have an [automatic installer](https://github.com/pyenv/pyenv-installer) that handles most things and talks you through the rest of the set up, but feel free to drop me a message if you run into any issues.